integration-tests/cypress/integration/api/devices/device_reregister.ts

```diff
@@ -0,0 +1,166 @@
+/// <reference path="../../../support/index.d.ts" />
+
+const HTTP_AuthorizationError = 401;
+const HTTP_BadRequest = 400;
+const HTTP_OK = 200;
+
+import { getTestName } from "../../../commands/names";
+import { getCreds } from "../../../commands/server";
+
+describe("Device reregister", () => {
+  const KEEP_DEVICE_NAME = true;
+  const GENERATE_UNIQUE_NAME = false;
+  const GENERATE_PASSWORD = null;
```

Comment:
do these need to be ocnstants?

Reply:
Was asked to use this approach (rather than just pass true/false) in past code review - to make code 'human readable' without the need to look at the called function to see what the parameter does.  Happy to revert to past behaviour - if that's better. Need 3rd opinion!

Reply:
constants are definitely more readable than anonymous true/false args IMO.